RetailCoder.VBE/UI/Settings/GeneralSettings.xaml

```diff
@@ -215,6 +216,13 @@
                             </DataGridTemplateColumn>
                         </DataGrid.Columns>
                     </DataGrid>
+
+                    <Label Content="{Resx ResxName=Rubberduck.UI.RubberduckUI, Key=GeneralSettings_ExperimentalFeatures}" FontWeight="SemiBold" />
+                    <Label x:Name="SC_Warning" Margin="20,-5,0,0" Content="{Resx ResxName=Rubberduck.UI.RubberduckUI, Key=GeneralSettings_EnableSourceControlWarning}"
```

Comment:
I feel like it's unnecessary to hide the warning message on experimental features.

A general warning along the lines of: "Only touch this if you know what you're doing: Enabling and/or using features from this section may result in things breaking in unforseen and irrecoverable ways." seems more appropriate, especially if we're going to introduce more features of this kind